packages/next-on-pages/src/buildApplication/processVercelFunctions/dedupeEdgeFunctions.ts

```diff
@@ -193,7 +225,16 @@ async function buildFunctionFile(
 		return acc;
 	}, new Map<string, string>());
 
+	let chunkMapIdx = 0;
+	const chunksExportsMap = new Map<string, Set<string>>();
+
 	groupedImports.forEach((keys, path) => {
+		if (path.endsWith('.wasm')) {
+			// we don't need/want to apply any code transformation for wasm imports
+			functionImports += `import ${keys} from "${path}";`;
```

Comment:
Should this be `import {${keys}} from "${path}";`?

Reply:
Also should the ìmportPath`be used?

nit: it looks like `relativeImportPath`can be moved out of the loop?

Reply:
> Should this be `import {${keys}} from "${path}";`?

![Screenshot 2024-08-01 at 10 37 20](https://github.com/user-attachments/assets/d77574cd-4358-4e2a-bb7c-d601228b6e10)

I think this works as intended, the fact is that wasm have only a single default export (I think!)

I should definitely at least update the code to make this more explicit! 👍

Reply:
> Also should the ìmportPath`be used?

> nit: it looks like relativeImportPathcan be moved out of the loop?

Yes and yes 🙂 , awesome, thanks a lot for catching those! 🙏

Reply:
Regarding the import string, I've updated the code to make it more clear (https://github.com/cloudflare/next-on-pages/pull/836/commits/8a3686154991c7adeb91dbb30a33a39c661a7313) please let me know if you find this better 🙂